config/settings/production.py

```diff
@@ -13,3 +15,21 @@
 ]
 
 RQ = {"WORKER_CLASS": "metecho.rq_worker.ConnectionClosingHerokuWorker"}
+
+if REDIS_LOCATION.startswith("rediss://"):
+    # Fix Redis errors with Heroku self-signed certificates
```

Comment:
Ah thanks, this was on my to-do list.

Reply:
Great! I ran into it on another project a month or so ago, and figured with Heroku giving Redis v5 EOL status, now's a decent time to upgrade.